web/themes/custom/cloudy/src/css/_pages.scss

```diff
@@ -39,4 +39,11 @@ h1.homepage-title {
   h3 {
     @include type-style($cloudy-heading-5);
   }
-}
\ No newline at end of file
+}
+
+// Applies sizing to headings in footer.
+.cloudy-footer {
```

Comment:
@brittanysmart , this is what I was talking about to reduce the number of block overrides. If we avoid applying the `class="h6"` style utility classes, and use a simple cascade. We can style every header in the footer with 4 lines of code rather than adding a block template or a hardcoded header to the page template.

This creates a smaller PR that is much easier to read and let's developers add blocks without having to do theming.

We need to move towards this approach on blocks and fields to make it easier to use Drupal to do what it does best.